sharktank/sharktank/ops/default_impls.py

```diff
@@ -27,6 +27,47 @@
 import iree.turbine.ops.iree
 
 
+@argmax.override(Tensor)
+def argmax_default(
+    x: Tensor,
+    dim: Optional[int] = None,
+    chunk_size: Optional[int] = None,
+    keepdim: bool = False,
```

Comment:
These two should be flipped. `chunk_size` is not an argument for `torch.argmax`, and may lead to silent bugs if people call `ops.argmax` expecting positional arguments to be in the same order as `torch.argmax` if torch will accept a bool instead of an int. Same idea for other signatures for this function.